styleguide/component.stories.js

```diff
@@ -75,12 +75,14 @@ export default {
     },
   },
   decorators: [
-    (storyFn) => `<div
-      class="cads-styleguide__language-label"
-      onclick="javascript:styleguideChangeLangugage();"
-      aria-hidden="true"
-    ></div>
-    ${storyFn()}`,
+    (storyFn) => `<div class="cads-styleguide__wrapper">
```

Comment:
Kept `cads-styleguide__wrapper` in the old component stories as it's used as a hook for some of the backstop tests. No longer any styling hooked to it though.